src/rebase-snap/src/main.rs

```diff
@@ -118,14 +140,16 @@ fn rebase(base_file: &mut File, diff_file: &mut File) -> Result<(), Error> {
     Ok(())
 }
 
-fn main() {
+fn main() -> Result<(), RsExitError> {
```

Comment:
This should return a custom error type (same comment as on `run_with_api` https://github.com/firecracker-microvm/firecracker/pull/3575/commits/f83b7c8cd3fdb3b320ee1fe1f80769c4c498ee56#r1163827837).